git_stats.py

```diff
@@ -0,0 +1,17 @@
+#! /bin/env python
+
+import sys
+data = sys.stdin.read()
+data = [k.split('\n\n') for k in [i.strip() for i in data.split('COMMITBEGIN')] if k]
```

Comment:
a little bit hard to read and understand what is going on here

Reply:
will improve